SwiftDiffusion/Views/MainViews/PromptView/OnChangeOfCurrentPrompt.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  OnChangeOfCurrentPrompt.swift
+//  SwiftDiffusion
+//
+//  Created by Justin Bush on 2/25/24.
+//
+
+import SwiftUI
+
+// TODO: MAJOR REFACTOR
+struct OnChangeOfCurrentPrompt: View {
```

Comment:
Will need to figure out different method of implementation for diff changes and/or on prompt change, save last prompt, etc.